### PR TITLE
Added validation

### PR DESCRIPTION
**Pull request by varundhand**
Please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,21 +56,21 @@
...
       </div>
-      <div class="box2 p-5 border-start">
+      <div class="box2 p-5 border-start w-100">
         <div class="form">
           <textarea
             readonly
             class="form-control p-0 response-box text-secondary"
             name="response"
-            style="height: 600px; width: 650px"
+            style="height: 600px; width: 100%;"
```
Comment (Vaasu-Dhand) [inline index.html]: Use bootstrap class `w-100` for consistency's sake.

File: script.js [hunk 2]
```diff
@@ -60,3 +60,23 @@ function showResponse(object) {
   console.log(responseElement);
   responseElement.value = object;
 }
+
+// Validation for page number keyboard input
+// TODO: Come back to this & find a better solution
```
Comment (Vaasu-Dhand) [inline script.js:65]: When would this be completed?

File: index.html [hunk 3]
```diff
@@ -56,21 +56,21 @@
...
-      <div class="box-1 p-5 border-end">
+      <div class="box-1 p-5 border-end w-100">
         <!--empty box-->
       </div>
-      <div class="box2 p-5 border-start">
+      <div class="box2 p-5 border-start w-100">
         <div class="form">
           <textarea
             readonly
             class="form-control p-0 response-box text-secondary"
```
Comment (Vaasu-Dhand) [inline index.html]:
```suggestion
            class="form-control p-0 response-box text-secondary w-100"
```

File: index.html (hunk 1 above)
Comment (Vaasu-Dhand) [inline index.html]:
```suggestion
            style="height: 600px;"
```

**Review by Vaasu-Dhand — CHANGES_REQUESTED** [review]
Looks good to me other than minor suggestions

**Comment by varundhand** [comment]
Addressed all the issues mentioned

**Review by Vaasu-Dhand — APPROVED** [review]

**Pull request merged**